src/containers/ApplyPage/index.js

```diff
@@ -0,0 +1,87 @@
+import React, { PureComponent } from 'react';
+import TSTextField from '../../components/TSTextField';
+import TSButton from '../../components/TSButton';
+import { Icon, Row, Col, Select } from 'antd';
```

Comment:
Did you install eslint already? If yes, then this should show error. Relative import should place after library import